Tasks/VsTestV2/Tests/package-lock.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@types/mocha": {
       "version": "5.2.5",
-      "resolved": "https://pkgs.dev.azure.com/mseng/PipelineTools/_packaging/PipelineTools_PublicPackages/npm/registry/@types/mocha/-/mocha-5.2.5.tgz",
```

Comment:
Any particular reason to move away from our secured feed to the public one?

Reply:
These got auto added on build

Reply:
Searching is rest of the code base, I see couple of places where official one is used. I guess it should be fine.